subworkflows/local/01_inspect_data/genome_properties.nf

```diff
@@ -21,7 +20,12 @@ workflow GENOME_PROPERTIES {
     GENESCOPEFK ( FASTK_HISTEX.out.hist )
 
     // Generate Smudgeplot
-    MERQURYFK_PLOIDYPLOT ( fastk_hist_ktab )
+    SMUDGEPLOT (
+        fastk_hist_ktab.map { meta, _hist, ktab -> [meta, ktab] },
+        // the genomescopefk kmer value = mean value of the heterozygous peak
+        // smudgeplot -L advice = avoid cutting off the monoploid peak, thus / 2
+        GENESCOPEFK.out.kmer_cov.map{ _meta, kmercov -> Math.round((kmercov as Double) / 2) as Integer }
```

Comment:
```suggestion
        GENESCOPEFK.out.kmer_cov.map{ _meta, kmercov -> (kmercov as Double) / 2).round() }
```

This number will likely never be high enough, but be careful using Integer as a type because it's 32-bit. This has caused issues before. `round()` should return a `Long`.